_components/form-controls/07-range-input.md

```diff
@@ -0,0 +1,40 @@
+---
+title: Range input
+parent: Form controls
+maturity: alpha
+order: 07
+lead: The range slider allows users to choose an approximate number from a range.
+---
+
+{% include code/preview.html component="range-input" %}
+{% include code/accordion.html component="range-input" %}
+
+<div class="usa-accordion-bordered component-docs">
+  <button class="usa-button-unstyled usa-accordion-button"
+      aria-expanded="true" aria-controls="range-docs">
+    Documentation
+  </button>
+  <div id="range-docs" aria-hidden="false" class="usa-accordion-content" markdown="1">
+#### Accessibility
+- As you customize this form template, ensure it continues to follow the [accessibility guidelines for form templates]({{ site.baseurl }}/form-templates/) and the [accessibility guidelines for form controls]({{ site.baseurl }}/form-controls/).
+
+#### Usability
+##### When to use
+- When you expect a user will move the slider back and forth to experiment with different inputs.
+- When the exact number a user enters isn’t important, but you want to give them an opportunity to pick from a range.
+
+##### When to consider something else 
+Use a regular text input if a user needs only to enter a precise number or only needs to enter a number once.
+
+##### Guidance
+- When a user clicks the slider control to change their input, the slider control should change color to indicate it is active.
+- Users should be able to either click somewhere along the slider or drag the circular control to the left or right to change the  value. 
+- Whenever appropriate, label each end of the slider with what it represents (whether a number range, “large” or “small,” “high” or “low”).
+- Set the `min` and `max` attribute of the `input` element to match whatever instructions or labels accompany the slider.
+- Set the `step` attribute so the slider is not too granular. Sliders should let users input approximate values. By setting the step to “10” or “20,” you keep users from inadvertently selecting values that are more precise than they intended.
```

Comment:
The guidance here only really works when the min is 0 and the max is 100. If the min/max were -1/1, this wouldn't help as much. Maybe it's more like `10% of the total range`?

> In a range slider, the relative value is more important than the specific value, so set the `step` attribute so it's not too granular. By setting the step to a value of 10-20% of the total range you prevent unnecessary precision and cognitive strain in your users.

Reply:
Updated in a42ccaf2c413889c5349f08d2e60202141531579.